### ISSUE TITLE
[DB] Add member points to member view

### ISSUE THREAD

**Issue by mellophone** [CougarCS/astro-api#7]
Aggregate function to reduce member point transaction table -> 1 contact: sum of mem points

Attach as field in member view

**Comment by mellophone** [comment]
So I first added a few entries to the member_point_transaction table as a quick test, and then I changed the member view code to left join with the member_point_transaction table based on contact_id. In order to use aggregate functions, I added the group by line so that the points will group based on the contact_id. Finally I just added the points field to the view and set it to the sum of each contact's points found in the member_point_transaction table. I used the coalesce() function so that instead of anyone's points being [NULL], it will just return as 0 instead.

Old member view code:
![Image](https://user-images.githubusercontent.com/54153434/177423174-eec3043c-c25b-4208-a09c-261578568bf3.png)

New member view code:
![Image](https://user-images.githubusercontent.com/54153434/177423026-08acc940-afd5-4ccc-a6d7-a987198f0a54.png)

Example member_point_transaction rows:
![Image](https://user-images.githubusercontent.com/54153434/177424097-9c6d56fa-f89f-4409-8e77-e982449049a5.png)

Member view results:
![Image](https://user-images.githubusercontent.com/54153434/177424407-606c0bc0-fb16-495b-a1dc-6764558bfc54.png)
